### DISCUSSION TITLE
Add Dropdown control to MAUI version

### DISCUSSION THREAD

**Discussion by jldearmas** [enisn/Xamarin.Forms.InputKit#364] [Ideas]
Is there any chance to copy Dropdown control from Xamarin version to the MAUI version?

Even, could be possible to create a new control DropDownMenu using the same PopupMenu effect?

**Comment by enisn** [comment]
On Android, it works with that popup effect. But unfortunately, I am stuck on the iOS implementation. Even in the official Apple documentation, they don't suggest using popup menus on phones, that effect is just for iPads and macbooks. So, implementing is very tricky
